src/components/calculator/Button.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Button(props) {
+    const { className, onClick, value } = props;
+
+    return (
+        <div className={className}>
+            <button onClick={onClick} value={value}>{value}</button>
+        </div>
+    )
+}
+
```

Comment:
Quando tens um componente simples que só retorna alguma coisa podes fazer assim:
```suggestion
const Button({ className, onClick, value }) => (
    <div className={className}>
        <button onClick={onClick} value={value}>{value}</button>
    </div>
 )

```